en/installation/installation-of-centreon-ha/overview.md

```diff
@@ -0,0 +1,167 @@
+
+### MariaDB functional group
+
+The `ms_mysql` functional group is a _multi-state_ resource. This resource can be in active/primary mode 
+on one node and in secondary/passive mode on another node. 
+
+The `ms_mysql-master` meta-resource is assigned to the primary database. 
+
+### Centreon functional group
+
+The `centreon` functional group gathers all Centreon resources to manage them. 
+
+### Resources type description
+
+All these resources are described in the table below.
+
+| Name                    | Type                 | Description                                          |
+| ----------------------- | -------------------- | ---------------------------------------------------- |
+| `ms_mysql`              | multi-state resource | Handles `mysql` process and the data replication     |
+| `ms_mysql-master`       | location             | Set MariaDB Master server rule preference            |
+| `php7`                  | clone service        | FastCGI Process Manager service (`rh-php73-php-fpm`) |
+| `cbd_rrd`               | clone service        | Broker RRD service (`cbd`)                           |
+| `centreon`              | group                | Centreon "primitive services" group                  |
+| `vip`                   | primitive service    | VIP address for centreon                             |
+| `http`                  | primitive service    | Apache service (`httpd24-httpd`)                     |
+| `gorgone`               | primitive service    | Gorgone service (`gorgoned`)                         |
+| `centreon_central_sync` | primitive service    | Files synchronization service                        |
+| `cbd_central_broker`    | primitive service    | Central Broker service (`cbd-sql`)                   |
+| `centengine`            | primitive service    | Centreon-Engine service (`centengine`)               |
+| `centreontrapd`         | primitive service    | SNMP Traps management service (`centreontrapd`)      |
+| `snmptrapd`             | primitive service    | SNMP Traps listening service (`snmptrapd`)           |
+
+**Note:** The resources of the `centreon` group are started one after the other in the list order.
+
+### Resources constraints
+
+Pacemaker offers various types of constraints:
+* Location: where the resource should or shouldn't run.
+* Colocation: how resources behave to each other.
+
+For example, Centreon-HA uses location constraints to specify to Pacemaker that the database process 
+needs to be up on backend nodes but not on frontend nodes. 
+
+Regarding colocation constraints, they can ensure a Virtual IP sticks to the master nodes and/or role. 
+Therefore Users, Pollers, and Daemons constantly interact with the primary node. 
+
+### QDevice and votes
+
+The configuration of a qdevice is **mandatory** to avoid split-brain and other situations nobody wants 
+to face in a Cluster. The server with the `quorum-device` role aims to obtain an absolute majority in a vote 
```

Comment:
```suggestion
to face in a Cluster. The server with the `quorum-device role aims` to obtain an absolute majority in a vote 
```

Reply:
Hum, I'm not sure why this modification is required?